RecoLocalTracker/SiPixelClusterizer/python/SiPixelClusterizer_cfi.py

```diff
@@ -30,14 +30,15 @@
 
 # Need these until phase2 pixel templates are used
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
+from SimTracker.SiPhase2Digitizer.phase2TrackerDigitizer_cfi import PixelDigitizerAlgorithmCommon
 phase2_tracker.toModify(siPixelClusters, # FIXME
   src = 'simSiPixelDigis:Pixel',
   MissCalibrate = False,
   Phase2Calibration = True,
-  Phase2ReadoutMode = -1, # Flag to decide Readout Mode : linear TDR (-1), dual slope with slope parameters (+1,+2,+3,+4 ...) with threshold subtraction
-  Phase2DigiBaseline = 1200.,
+  Phase2ReadoutMode = PixelDigitizerAlgorithmCommon.Phase2ReadoutMode, # Flag to decide Readout Mode : linear TDR (-1), dual slope with slope parameters (+1,+2,+3,+4 ...) with threshold subtraction
```

Comment:
to be extremely picky (an inline with the current good practices as seen in recent RECO PRs) one could drop the type specification and use  `PixelDigitizerAlgorithmCommon.Phase2ReadoutMode.value()`

Reply:
It's not only about the type specification. Configuration like
```python
phase2_tracker.toModify(siPixelClusters,
    ...
    Phase2ReadoutMode = PixelDigitizerAlgorithmCommon.Phase2ReadoutMode
    ...
)
```
would make `siPixelClusters.Phase2ReadoutMode` to be exactly the same `cms.int32` object as `PixelDigitizerAlgorithmCommon.Phase2ReadoutMode`, which means that if anyone would change the value of `PixelDigitizerAlgorithmCommon.Phase2ReadoutMode` later in the python program execution without "specifying the type", also `siPixelClusters.Phase2ReadoutMode` would change automatically (being the same object). Example code:
```python
from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
process = cms.Process("TEST", phase2_tracker)
process.load("RecoLocalTracker.SiPixelClusterizer.SiPixelClusterizer_cfi")

from SimTracker.SiPhase2Digitizer.phase2TrackerDigitizer_cfi import PixelDigitizerAlgorithmCommon
PixelDigitizerAlgorithmCommon.Phase2ReadoutMode = 5
# now siPixelClusters.Phase2ReadoutMode == 5
```
Usually this kind of "long distance interaction" can lead to very confusing behavior, which is why core recommends to always `clone()`/`copy()`/`value()` the value of `cms.<type>` when assigning from an existing configuration object to a new one.